libraries/stdlib/src/kotlin/random/Random.kt

```diff
@@ -290,7 +295,11 @@ public abstract class Random {
 
         override fun nextBytes(array: ByteArray): ByteArray = defaultRandom.nextBytes(array)
         override fun nextBytes(size: Int): ByteArray = defaultRandom.nextBytes(size)
-        override fun nextBytes(array: ByteArray, fromIndex: Int, toIndex: Int): ByteArray = defaultRandom.nextBytes(array, fromIndex, toIndex)
+        override fun nextBytes(array: ByteArray, fromIndex: Int, toIndex: Int): ByteArray = defaultRandom.nextBytes(
+            array,
+            fromIndex,
+            toIndex
+        )
```

Comment:
This change is not related. Better to keep it consistent with the other function definitions.